### DISCUSSION TITLE
Cannot destructure property `inAmpMode` of 'undefined' or 'null'.

### DISCUSSION THREAD

**Discussion by Yangchunyu01** [vercel/next.js#14114] [Help]
Everything is well when develop on Windows environment,
But got this error, after publish to linux server, when running "npm run dev" command

**Comment by iraizo** [comment]
i get the same thing

**Comment by iraizo** [comment]
@Yangchunyu01 This is an actual issue with your react code, not next.js i changed everything to a function component and it started working

**Comment by IamSAL** [comment]
If this is occurring in a monorepo, make sure only the root package.json and node_modules contains nextjs.

This issue occurs when you have inner packages/apps separate version of NextJs installed with their own package.json/node_modules.
